test/gameSetups/testGameTemplate.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Wondering if maybe the example setup should be a little more complex so people know what all the configuration options are. I can send an example one with more stuff going on if that would help

Reply:
Yeah, was thinking the same. An example that touched all options would be good to have as the default.